pkgs/tools/graphics/obj2voxel/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, static ? stdenv.hostPlatform.isStatic
+, config
+, cmake
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "obj2voxel";
+  version = "1.3.4";
+
+  src = (
+    fetchFromGitHub {
+      owner = "Eisenwave";
+      repo = finalAttrs.pname;
+      rev = "refs/tags/v${finalAttrs.version}";
+      fetchSubmodules = true;
+      hash = "sha256-FkGQjYm15le1Na+2U1TgK4nNOZNs5bME5J87pjVFpDs=";
+    }
+  ).overrideAttrs (
+    _: {
+      GIT_CONFIG_COUNT = 1;
+      GIT_CONFIG_KEY_0 = "url.https://github.com/.insteadOf";
+      GIT_CONFIG_VALUE_0 = "git@github.com:";
+    }
+  );
+
+  patches = [
+    # pull missing definitions (https://github.com/Eisenwave/voxel-io/pull/3)
+    ./stdlib-defs.patch
+  ];
+
+  postPatch = ''
+    echo 'install(TARGETS obj2voxel-cli DESTINATION ''${CMAKE_INSTALL_PREFIX}/bin)' >> CMakeLists.txt
+  '';
+
+  cmakeFlags = [
+    "-DBUILD_SHARED_LIBS=${if static then "OFF" else "ON"}"
+  ];
+
+  strictDeps = true;
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ ];
+
+  meta = with lib; {
+    description = "Convert OBJ and STL files to voxels, with support for textures.";
```

```suggestion
    description = "Convert OBJ and STL files to voxels, with support for textures";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes